### PR TITLE
Update backup docs

### PR DESCRIPTION
**Pull request by ggezer**

### CODE REVIEW

File: docs/en/operation/Journal.md [hunk 1]
```diff
@@ -158,6 +159,27 @@ the default backup directory would be `hdfs://192.168.1.1:9000/alluxio_backups`.
 The files to retain in the backup directory is limited by `alluxio.master.daily.backup.files.retained`.
 Users can set this property to the number of backup files they want to keep in the backup directory.
 
+### Backup delegation on HA cluster
+Alluxio supports taking backup without causing service unavailability on a HA cluster configuration.
+When enabled, Alluxio leader master delegates backups to standby masters in cluster.
```
Comment (LuQQiu) [inline docs/en/operation/Journal.md]: I just find that in our doc convention, we use leading master https://docs.alluxio.io/os/user/edge/en/contributor/Documentation-Conventions.html.

File: docs/en/operation/Journal.md [hunk 2]
```diff
@@ -158,6 +159,27 @@ the default backup directory would be `hdfs://192.168.1.1:9000/alluxio_backups`.
...
+After configuring backup delegation, both manual and scheduled backups will run in delegated mode.
+
+Backup delegation can be configured with below properties:
+- `alluxio.master.backup.delegation.enabled`: Whether to delegate backups to stand-by masters. Default: `false`.
+- `alluxio.master.backup.heartbeat.interval`: Interval at which stand-by master that is taking the backup will update the leader master with status. Default: `1sec`.
```
Comment (LuQQiu) [inline docs/en/operation/Journal.md]: with backup process status?

File: docs/en/operation/Journal.md [hunk 3]
```diff
@@ -158,6 +159,27 @@ the default backup directory would be `hdfs://192.168.1.1:9000/alluxio_backups`.
...
+
+There are some advanced properties that controls the communication between Alluxio masters for coordinating the backup:
+- `alluxio.master.backup.transport.timeout`: Communication timeout for messaging between masters for coordinating backup. Default: `5sec`.
+- `alluxio.master.backup.connect.interval.min`: Minimum duration to sleep before retrying after unsuccessful handshake between stand-by master with leader master. Default: `1sec`.
+- `alluxio.master.backup.connect.interval.max`: Maximum duration to sleep before retrying after unsuccessful handshake between stand-by master with leader master. Default: `10sec`.
+- `alluxio.master.backup.abandon.timeout`: Specifies how long the leader master waits for a heart-beat before abandoning the backup. Default: `2min`.
+
+
```
Comment (LuQQiu) [inline docs/en/operation/Journal.md]: remove one empty line?

**Comment by ggezer** [comment]
alluxio-bot, merge this please

**Pull request merged**